web/src/ResourceList.js

```diff
@@ -28,12 +29,21 @@ class ResourceSummary extends Component {
     let k8sStatus = getK8sStatus(resource)
     let buildStatus = getBuildStatus(resource)
     let updateTime = getUpdateTime(resource)
+    let endpoint = getEndpoint(resource)
+
     return (
       <li role="rowgroup" className="resource">
         <p role="cell" className="cell name">{resource.Name}</p>
         <p role="cell" className="cell">{k8sStatus}</p>
         <p role="cell" className="cell">{buildStatus}</p>
         <p role="cell" className="cell">{updateTime}</p>
+        {endpoint
```

Comment:
for what it's worth, i often find this easier to read in a variable outside the template